e2e/utils.go

```diff
@@ -941,7 +941,7 @@ func writeDataInPod(app *v1.Pod, f *framework.Framework) error {
 	// write data to PVC
 	filePath := app.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
 
-	_, writeErr := execCommandInPod(f, fmt.Sprintf("dd if=/dev/zero of=%s bs=1M count=300 status=none", filePath), app.Namespace, &opt)
+	_, writeErr := execCommandInPod(f, fmt.Sprintf("dd if=/dev/zero of=%s bs=1M count=10 status=none", filePath), app.Namespace, &opt)
```

Comment:
Also changing the size of the write. Is that intentional? It is not mentioned in the commit message.

Reply:
This is now documented, so we are good.